apps/api/v2/src/ee/event-types/event-types_2024_04_15/outputs/get-event-type-public.output.ts

```diff
@@ -87,6 +87,14 @@ class BookingField {
   @Type(() => Source)
   @IsOptional()
   sources?: Source[];
+
+  @IsOptional()
+  @IsNumber()
+  maxLength?: number;
```

Comment:
Same as with inputs above - I think min and max length should be part of the individual booking field, because right now we are defining min and max length of an event type.